src/main/java/com/corndel/cashregister/Level2.java

```diff
@@ -14,15 +17,33 @@ public class Level2 {
    * N.b. $1 is a note, not a coin
    */
   public static int countCoins(List<Item> drawer) {
-    // TODO
-    return 0;
+    int count = 0;
+    ArrayList<String> coinList = new ArrayList<>(Arrays.asList("quarter", "dime", "nickel", "penny"));
+
+    for (Item item : drawer) {
```

Comment:
This is great makes lots of sense.

One tweak you could make is to use a more performant Data structure for lookups.

A Set is better than a List for this. Have a look at the Set interface and a Java implementation [HashSet](https://www.w3schools.com/java/java_hashset.asp).